### PR TITLE
London/8-Mahri-Atdayeva-JS-week-2

### PR DESCRIPTION
**Pull request by mahri09**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: mandatory/2-function-creation.js [hunk 1]
```diff
@@ -16,7 +18,17 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
```
Comment (DelroyGayle) [inline mandatory/2-function-creation.js:21]: Try testing some 1 decimal place numbers e.g. 3.1, 10.2 etc with num.toFixed(2)
It maybe that you do not need to use num.toFixed(1) at all!

**Review by DelroyGayle — COMMENTED** [review]
Great work, Mahri. Thank you.

I learnt a couple of new tricks looking at your code.
One thing, however, TA Jack Lee, discouraged  the use of arrow notation 
**const getAllFrequencies = () => etc**

He stated that this was _advanced usage_  of Javascript; and rather, CYF wants us to use **function () {}**

I thought I should point that out.

Thank you

File: exercises/G-conditionals-2/exercise-1.js [hunk 2]
```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
```
[the remark this answers is not in the document]
Comment (mahri09) [reply exercises/G-conditionals-2/exercise-1.js:9]: @DelroyGayle or we can say like this :  if (number < 0) {
    return "negative";
  } else if (number >= 0) {
    return "positive";
  } else {
    return "it is not a number!";
  }
  }

File: exercises/G-conditionals-2/exercise-4.js [hunk 3]
```diff
@@ -8,7 +8,9 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  return sentence.includes("code");
```
[the remark this answers is not in the document]
Comment (mahri09) [reply exercises/G-conditionals-2/exercise-4.js:12]: @DelroyGayle , here you can find some links  examples that we can use find word in sentence with include() method : https://www.geeksforgeeks.org/javascript-string-includes-method/
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

**Comment by mahri09** [comment]
> 

@DelroyGayle thanks for reviewing and I missed this conversation didn't now that we shouldn't use arrow functions it was good you mentioned it!

File: exercises/G-conditionals-2/exercise-4.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (DelroyGayle) [reply exercises/G-conditionals-2/exercise-4.js:12]: I stand corrected. Apologies.

[the remark this answers is not in the document]
Comment (mahri09) [reply exercises/G-conditionals-2/exercise-4.js:12]: @DelroyGayle no worries, we learning altogether, thank you for reviewing and spending time on my code I did correct my mistakes thank you so much!

**Pull request closed without merging**